CommonItems/ConnectionBox.qml

```diff
@@ -49,12 +49,13 @@ Rectangle {
         Label { text: "IP:" }
         TextField {
             id: ip
-
+            selectByMouse: true
         }
 
         Label { text: "Port:"}
         TextField {
             id: port
+            selectByMouse: true
             validator: IntValidator{}
         }
```

Comment:
((GENERAL))

default localhost ip and default port could be already typed as default, but it can be out of scope of this PR

Reply:
fixed